pkg/satellite/pass/list_passes.go

```diff
@@ -36,8 +36,12 @@ var listPassesVerboseTemplate = []printer.TemplateItem{
 	{"MAX_ELEVATION_TIME", "maxElevationTime"},
 	{"CHANNEL_SET_ID", "channelSet.id"},
 	{"CHANNEL_SET_NAME", "channelSet.name"},
-	{"DL_FREQ_HZ", "downlinkFrequency"},
-	{"UL_FREQ_HZ", "uplinkFrequency"},
+	{"DL_FREQ_HZ", "channelSet.downlink.frequency"},
+	{"DL_MODULATION", "channelSet.downlink.modulation"},
+	{"DL_BITRATE", "channelSet.downlink.bitrate"},
+	{"UL_FREQ_HZ", "channelSet.uplink.frequency"},
```

Comment:
If `channelSet.uplink` doesn't exist, will we get a NPE when trying to access modulation, bitrate, and frequency?

Reply:
That's very good catch. I am going to fix it in a separate PR. Thanks!

Reply:
Ref. https://github.com/golang/go/issues/32231